From f6485ff7c2d083e33c4229b4abb24c03366e22f5 Mon Sep 17 00:00:00 2001
From: Luiz Augusto von Dentz <luiz.dentz-von@nokia.com>
Date: Thu, 2 Dec 2010 10:40:33 +0200
Subject: [PATCH] Fix deinitializing telephony backend when it wasn't initialized

Telephony driver is now deinitialized on headset adapter driver remove so
telephony_exit should not be called on plugin exit anymore.
---
 audio/manager.c |    4 +---
 1 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/audio/manager.c b/audio/manager.c
index 12671c7..035656b 100644
--- a/audio/manager.c
+++ b/audio/manager.c
@@ -1264,10 +1264,8 @@ void audio_manager_exit(void)
 	if (enabled.media)
 		btd_unregister_adapter_driver(&media_server_driver);
 
-	if (enabled.headset) {
+	if (enabled.headset)
 		btd_unregister_adapter_driver(&headset_server_driver);
-		telephony_exit();
-	}
 
 	if (enabled.gateway)
 		btd_unregister_adapter_driver(&gateway_server_driver);
-- 
1.7.1

